src/main/scala/atom/Atom.scala

```diff
@@ -9,6 +9,39 @@ trait Disposable extends js.Object {
   def dispose(): Unit = js.native
 }
 
+/**
+ *  An object that aggregates multiple Disposable instances together into a
+ *  single disposable, so they can all be disposed as a group.
+ */
+@js.native
+trait CompositeDisposable extends Disposable {
```

Comment:
What is the profit of extending `Disposable` if you still need to override the only inherited method?

Reply:
I just follow the API like in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/atom/index.d.ts. I can remove the `extends` if you want.

Reply:
Well, they have a common trait `DisposableLike` there... But don't mind, I think it doesn't matter here much.